save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/DownloadFilesController.kt

```diff
@@ -210,6 +206,42 @@ class DownloadFilesController(
         }
     }
 
+    private fun getExecutionId(testExecutionDto: TestExecutionDto): Long {
+        testExecutionDto.executionId?.let { return it }
```

Comment:
why `executionId` can be null for `testExecutionDto`?

Reply:
because then we build debugInfo to TestExecutionDto( on SaveAgent it can't bind it to executionId yet because it doesn't have access to the database

Reply:
can we add executionId to debugInfo then?